inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -396,21 +416,15 @@ def refextract(obj, eng):
         return
 
     pdf_references, text_references = [], []
-    source = get_value(obj.data, 'acquisition_source.source')
+    source = get_source(obj.data)
 
     with get_document_in_workflow(obj) as tmp_document:
         if tmp_document:
-            try:
-                pdf_references = extract_references_from_pdf(tmp_document, source)
-            except TimeoutError:
-                obj.log.error('Timeout when extracting references from PDF.')
+            pdf_references = extract_references_from_pdf(tmp_document, source)
```

Comment:
This prevents this worthless log from happening: https://sentry.inspirehep.net/inspire-sentry/prod/issues/358/.